lib/retry.c

```diff
@@ -62,7 +63,9 @@ EXPORTED ssize_t retry_read(int fd, void *vbuf, size_t nbyte)
     char *buf = vbuf;
 
     for (nread = 0; nread < nbyte; ) {
+        syslog(LOG_DEBUG, "%s: top of loop, nread=%lu", __func__, (uint64_t) nread);
         ssize_t n = read(fd, buf + nread, nbyte - nread);
+        syslog(LOG_DEBUG, "%s: read returned %ld (%m)", __func__, (int64_t) n);
```

Comment:
For sure these should be removed before we ship this - or at least hidden behind an `#ifdef` so we don't have to call out to any additional syscalls during such a tight loop!  retry_read is used a lot.